script/deploy/healthCheck.ts

```diff
@@ -260,22 +261,22 @@ const main = defineCommand({
       })
       const approvedDexs = await dexManager.read.approvedDexs()
 
-      // Loop through dexs excluding the address for FeeCollector, LiFuelFeeCollector and TokenWrapper
+      // Loop through DEXs excluding the address for FeeCollector, LiFiDEXAggregator and TokenWrapper
       let numMissing = 0
       for (const dex of dexs.filter(
         (d) => !corePeriphery.includes(getAddress(d))
       )) {
         if (!approvedDexs.includes(getAddress(dex))) {
-          logError(`Dex ${dex} not approved in Diamond`)
+          logError(`DEX ${dex} not approved in Diamond`)
           numMissing++
         }
       }
 
-      // Check that FeeCollector, LiFuelFeeCollector and TokenWrapper are included in approvedDexs
+      // Check that FeeCollector, LiFiDEXAggregator and TokenWrapper are included in approvedDexs
       const feeCollectors = corePeriphery.filter(
```

Comment:
maybe would be good to change variable name (`feeCollectors`) as there are no only fee collectors but also dex aggregator?

Reply:
updated